packmanager/umbrella.go

```diff
@@ -123,6 +126,55 @@ func (u umbrella) AddSource(ctx context.Context, source string) error {
 	return nil
 }
 
+func (u umbrella) Prune(ctx context.Context, qopts ...QueryOption) error {
+	var query Query
+	var tarFiles []fs.DirEntry
+	var errList []error
+	homeDir, err := os.UserHomeDir()
+	sourcesDir := homeDir + "/.local/share/kraftkit/sources/"
```

Comment:
The sources directory can be calculated from the user-provided configuration at:
```
config.G[config.KraftKit](ctx).Paths.Sources
```

Reply:
Done 👍🏽